client/extensions/woocommerce/woocommerce-services/views/shipping-label/label-purchase-modal/sidebar.js

```diff
@@ -57,13 +59,24 @@ export const Sidebar = props => {
 	};
 	const onPaperSizeChange = value => props.updatePaperSize( orderId, siteId, value );
 
+	const hasSelectedRate = hasSelectedRates( form.rates );
+	let labelRequiresPaymentMethod = false;
+	if( hasSelectedRate ) {
+		forEach( form.rates.values , ( label ) => {
+			if( 'ups' !== label.carrierId ) {
```

Comment:
I think this condition can be merged with the variable declaration above.

Reply:
Sorry for my confusion, would you be able to expand on this? I was definitely looking to condense this bit here if possible.

Reply:
I think I got this cleaned up and put the conditions right into the assignment now. If you wouldn't mind taking a look at the update and let me know if that makes sense to do it that way?

Reply:
yeah, sorry Brian, my explanation was very poor. That was exactly what I meant :smile: